ctfpad/templates/ctfpad/challenges/import.html

```diff
@@ -0,0 +1,104 @@
+                        {% csrf_token %}
+                        <div class="modal-body">
+                            <div class="form-group">
+                                <input type="hidden" id="{{ form.ctf.id_for_label }}" name="{{ form.ctf.html_name }}"
+                                       value="{{ form.ctf.value }}"/>
+
+                                <label for="{{ form.format.id_for_label }}"
+                                       class="label"><strong>Format</strong></label>
+                                <div class="input-group mb-3">
+                                    <select id="{{ form.format.id_for_label }}" name="{{ form.format.html_name }}"
+                                            class="form-control">
+                                        {% for choice in form.format.field.choices %}
+                                            <option value="{{ choice.0 }}">{{ choice.1 }}</option>
+                                        {% endfor %}
+                                    </select>
+                                </div>
+
+                                <label for="{{ form.data.id_for_label }}" class="label"><strong>Data</strong></label>
+                                <div class="input-group mb-3">
+                                    <div class="input-group-append">
+                                        <span class="input-group-text">
+                                            <i class="fas fa-file-import"></i>
+                                        </span>
+                                    </div>
+                                    <textarea id="{{ form.data.id_for_label }}"
+                                              name="{{ form.data.html_name }}"
+                                              placeholder="Data"
+                                              class="form-control"
+                                              required>{% if form.data.value %}
+                                        {{ form.data.value }}{% endif %}</textarea>
+                                </div>
+                            </div>
+
+                            <div class="card-footer text-muted">
+                                <div class="control card-footer-item">
+                                    <button type="button" class="btn-primary btn-sm btn-block"
+                                            onclick="this.form.submit();">Import Challenges
+                                    </button>
+                                    <button type="button" class="btn btn-secondary btn-sm btn-block"
+                                            onclick="window.history.back();">Cancel
+                                    </button>
+                                </div>
+                            </div>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </div>
+    <script>
+        window.addEventListener('DOMContentLoaded', (event) => {
```

Comment:
Can you put this instead in a function in a script located in the `static/js`? Having bricks of JS everywhere makes it hard to track

Reply:
Done.